packages/admin-panel/src/pages/resources/TreeResourcePage.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { HorizontalTree } from '@tupaia/ui-components';
+import { Header, PageBody } from '../../widgets';
+import { usePortalWithCallback } from '../../utilities';
+import { LogsModal } from '../../logsTable';
+import * as COLORS from '../../theme/colors';
+import { useApi } from '../../utilities/ApiProvider';
+
+const Container = styled(PageBody)`
+  overflow: auto;
+`;
+
+const StyledHorizontalTree = styled(HorizontalTree)`
+  margin-top: 40px;
+  margin-bottom: 40px;
+  max-height: 870px;
```

Comment:
This height was chosen so that 20 items will be shown without having to use the component's scrollbar